hnn_core/tests/test_optimization.py

```diff
@@ -113,15 +117,26 @@ def test_optimize_evoked():
                        'any evoked drives'):
         net_empty = net_offset.copy()
         del net_empty.external_drives['evprox1']
+        del net_empty.external_drives['evdist1']
         net_opt = optimize_evoked(net_empty, tstop=tstop, n_trials=n_trials,
                                   target_dpl=dpl_orig, initial_dpl=dpl_offset)
 
+    with pytest.raises(ValueError, match='The drives selected to be optimized '
+                       'are not evoked drives'):
+        net_testBursty = net_offset.copy()
+        which_drives = ['bursty1']
+        net_opt = optimize_evoked(net_testBursty, tstop=tstop,
```

Comment:
```suggestion
        net_opt = optimize_evoked(net_test_bursty, tstop=tstop,
```

snake_case, snake_case :)